packages/expo-image/ios/ResizedAnimatedCoder.swift

```diff
@@ -0,0 +1,49 @@
+// Copyright 2024-present 650 Industries. All rights reserved.
+
+import SDWebImage
+
+/**
+ An animated image resizer that adjusts an already loaded `SDAnimatedImage`.
+ This is not a formal SDWebImage coder; it's used solely to leverage the SDWebImage loading pipeline.
+ */
+internal final class ResizedAnimatedCoder: SDImageIOAnimatedCoder {
+  private let image: SDAnimatedImage
+  private let size: CGSize
+  private let scale: Double
+
+  @available(*, unavailable)
+  required init(incrementalWithOptions options: [SDImageCoderOption: Any]? = nil) {
+    fatalError("Unsupported initializer")
+  }
+
+  @available(*, unavailable)
+  required init(animatedImageData: Data?, options: [SDImageCoderOption: Any]? = nil) {
+    fatalError("Unsupported initializer")
+  }
+
+  init(image: SDAnimatedImage, size: CGSize, scale: Double) {
+    self.image = image
+    self.size = size
+    self.scale = scale
+    super.init()
+  }
+
+  override func animatedImageFrame(at index: UInt) -> UIImage? {
+    guard let frame = image.animatedImageFrame(at: index) else {
```

Comment:
Change this into `super.animatedImageFrame(at: index)`, no need any `SDAnimatedImage` instance

Reply:
same as above the `super` is unavailable because we didn't decode